specs/protocol/isthmus/configurability.md

```diff
@@ -33,30 +35,30 @@ configurability.
 
 The `ConfigType` enum represents configuration that can be modified.
 
-| Name | Value | Description |
-| ---- | ----- | --- |
-| `GAS_PAYING_TOKEN` | `uint8(0)` | Modifies the gas paying token for the chain |
-| `BASE_FEE_VAULT_CONFIG` | `uint8(1)` | Sets the Fee Vault Config for the `BaseFeeVault` |
-| `L1_FEE_VAULT_CONFIG` | `uint8(2)` | Sets the Fee Vault Config for the `L1FeeVault` |
-| `SEQUENCER_FEE_VAULT_CONFIG` | `uint8(3)` | Sets the Fee Vault Config for the `SequencerFeeVault` |
-| `L1_CROSS_DOMAIN_MESSENGER_ADDRESS` | `uint8(4)` | Sets the `L1CrossDomainMessenger` address |
-| `L1_ERC_721_BRIDGE_ADDRESS` | `uint8(5)` | Sets the `L1ERC721Bridge` address |
-| `L1_STANDARD_BRIDGE_ADDRESS` | `uint8(6)` | Sets the `L1StandardBridge` address |
-| `REMOTE_CHAIN_ID` | `uint8(7)` | Sets the chain id of the base chain |
+| Name                                | Value      | Description                                           |
+| ----------------------------------- | ---------- | ----------------------------------------------------- |
+| `GAS_PAYING_TOKEN`                  | `uint8(0)` | Modifies the gas paying token for the chain           |
+| `BASE_FEE_VAULT_CONFIG`             | `uint8(1)` | Sets the Fee Vault Config for the `BaseFeeVault`      |
+| `L1_FEE_VAULT_CONFIG`               | `uint8(2)` | Sets the Fee Vault Config for the `L1FeeVault`        |
+| `SEQUENCER_FEE_VAULT_CONFIG`        | `uint8(3)` | Sets the Fee Vault Config for the `SequencerFeeVault` |
+| `L1_CROSS_DOMAIN_MESSENGER_ADDRESS` | `uint8(4)` | Sets the `L1CrossDomainMessenger` address             |
+| `L1_ERC_721_BRIDGE_ADDRESS`         | `uint8(5)` | Sets the `L1ERC721Bridge` address                     |
+| `L1_STANDARD_BRIDGE_ADDRESS`        | `uint8(6)` | Sets the `L1StandardBridge` address                   |
+| `REMOTE_CHAIN_ID`                   | `uint8(7)` | Sets the chain id of the base chain                   |
 
 ## `SystemConfig`
 
 ### `ConfigUpdate`
 
 The following `ConfigUpdate` event is defined where the `CONFIG_VERSION` is `uint256(0)`:
 
-| Name | Value | Definition | Usage |
-| ---- | ----- | --- | -- |
-| `BATCHER` | `uint8(0)` | `abi.encode(address)` | Modifies the account that is authorized to progress the safe chain |
-| `FEE_SCALARS` | `uint8(1)` | `(uint256(0x01) << 248) \| (uint256(_blobbasefeeScalar) << 32) \| _basefeeScalar` | Modifies the fee scalars |
-| `GAS_LIMIT` | `uint8(2)` | `abi.encode(uint64 _gasLimit)` | Modifies the L2 gas limit |
-| `UNSAFE_BLOCK_SIGNER` | `uint8(3)` | `abi.encode(address)` | Modifies the account that is authorized to progress the unsafe chain |
-| `EIP_1559_PARAMS` | `uint8(4)` | `uint256(uint64(uint32(_denominator))) << 32 \| uint64(uint32(_elasticity))` | Modifies the EIP-1559 denominator and elasticity |
+| Name                  | Value      | Definition                                                                        | Usage                                                                |
```

Comment:
Missing `FEE_VAULT_ADMIN` which is added here https://github.com/defi-wonderland/optimism/pull/274

Reply:
There will be a need for addition on the `Initialization` paragraph aswell probably